app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-<h1> Welcome to the Post page with list of users posts </h1>
\ No newline at end of file
+<h1> List of Users Posts </h1>
+
+<section>
+      <div>
+        <img src='<%= @users.photo %>' alt= '<%= @users.name %> photo' />
+      </div>
+      <span>
+        <%= @users.name %>
+        <p>Number of posts: <%= @users.posts_counter %> </p>
+      </span>
+      <div>
+        <%= @users.posts.each do |post| %>
+        <span>
+        <h5><%= link_to post.title, user_post_path(@users, post) %></h5>
+        <p> <%= post.text %> </p>
+        </span>
+        <span>
+          Comments: <%= post.comments_counter || 0 %>
+          Likes: <%= post.likes_counter || 0 %>
+        </span>
+        <div>
+          <ul>
+            <% if post.recent_comments.length == 0 %>
+            <p>No comments</p>
+            <% else %>
+              <% post.recent_comments.each do |comment| %>
+                <li><%= comment.author %> : <%= comment.text %></li>
+                <% end %>
```

Comment:
- [ ] [OPTIONAL] To retrieve the author's name, you can do `comment.author.name`, currently, the app shows an object of that class.

Reply:
Thank you, I will update the changes :man_technologist: